martin/src/srv/tiles.rs

```diff
@@ -183,10 +196,17 @@ impl<'a> DynTileSource<'a> {
                     tile = decode(tile)?;
                 }
             }
+
             if tile.info.encoding == Encoding::Uncompressed {
+                let ordered_encodings = match self.preferred_enc {
+                    Some(PreferredEncoding::Gzip) => PREFER_GZIP_ENC,
+                    Some(PreferredEncoding::Brotli) | None => PREFER_BROTLI_ENC,
+                };
+
                 // only apply compression if the content supports it
                 if let Some(HeaderEnc::Known(enc)) =
-                    accept_enc.negotiate(SUPPORTED_ENCODINGS.iter())
+                    // accept_enc.negotiate(SUPPORTED_ENCODINGS.iter())
```

Comment:
```suggestion
```

Reply:
Our description is  too verbose.
"Preferred tile encoding when martin is negotiating with browser, only works if the tile itself is not pre-compressed. The default Brotli is much smaller(and may be faster with caching) but a little bit slower than gzip."